pinot-core/src/main/java/org/apache/pinot/core/data/manager/config/TableDataManagerConfig.java

```diff
@@ -35,6 +35,7 @@
   private static final String TABLE_DATA_MANAGER_DATA_DIRECTORY = "directory";
   private static final String TABLE_DATA_MANAGER_CONSUMER_DIRECTORY = "consumerDirectory";
   private static final String TABLE_DATA_MANAGER_NAME = "name";
+  private static final String TABLE_IS_DIMENSION = "isDimTable";
```

Comment:
We should not have this config on instance level. It does not make sense for a server to only serve dimension table

Reply:
Sorry, I didn't quite get this, can you elaborate?

Reply:
The config in this class is for server specific config (the location of the data directory etc.). We should not keep the `isDimTable()` info in this class, it should be in table config only. Also, I don't think `isDimTable()` is used